src/DefaultProgressBar.jsx

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
```

Comment:
what are all these new components?

Reply:
These new components come from the internal fork of Rheostat

Reply:
These were copied over from the internal fork. I think it does make sense to include them in this change?

Reply:
@ljharb What do you mean?